### DISCUSSION TITLE
What happens if you connect more than one wire to one input of any component (logical/regex/arithmetic)?

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14925] [Ideas and suggestions]
Сomponents (and/or and many others) have 3 inputs. However, 15 wires can be connected to. What happens if you connect more than one wire to one input? How are the signals combined in them?
Does this happen in the same way in any type component or does it depend on the type of ?
I am especially interested in the "assign output value" input
If you connect 2 wires to it and send 100 and -100 at the same time, what will be the output?
How does signal combining/competition occur if several wires are connected to 1 input?
Of course, I can assemble the circuit and look at it with electrician's glasses. 
And I will definitely do this. But I am afraid of missing something important.
I would like to know the theory.

**Comment by Regalis11** [comment]
There's an explanation of how it works here: https://github.com/FakeFishGames/Barotrauma/issues/12993

In short, it depends on the type of the component, most components just use the last received value though. In general, you should not be connecting multiple wires to the same input: that in a sense means you're giving the component conflicting instructions, and the end result can be unpredictable.
